geopandas/sindex.py

```diff
@@ -839,13 +847,23 @@ def query_bulk(self, geometry, predicate=None, sort=False):
 
         @doc(BaseSpatialIndex.nearest)
         def nearest(
-            self, geometry, return_all=True, max_distance=None, return_distance=False
+            self,
+            geometry,
+            return_all=True,
+            max_distance=None,
+            return_distance=False,
+            exclusive=False,
         ):
             if not (compat.USE_SHAPELY_20 or compat.PYGEOS_GE_010):
                 raise NotImplementedError(
                     "sindex.nearest requires shapely >= 2.0 or pygeos >= 0.10"
                 )
 
+            if (exclusive) and (not compat.USE_SHAPELY_20):
```

Comment:
```suggestion
            if exclusive and not compat.USE_SHAPELY_20:
```

Reply:
@harisbal was there a reason to retain the parens?  This was marked as resolved but not changed...

Reply:
sorry I think black autofixed that